src/features/apiDelegation/offered/ChooseOrgPage/ChooseOrgPage.tsx

```diff
@@ -110,147 +107,92 @@ export const ChooseOrgPage = () => {
     }
   }
 
-  const infoPanel = () => {
-    if (reporteeOrgNumber === searchString && searchString.length > 0) {
-      return (
-        <Alert
-          severity='warning'
-          role='alert'
-        >
-          <Heading
-            size={'xsmall'}
-            level={2}
-            spacing
-          >
-            {t('api_delegation.own_orgnumber_delegation_heading')}
-          </Heading>
-          <Paragraph>{t('api_delegation.own_orgnumber_delegation_paragraph')}</Paragraph>
-        </Alert>
-      );
-    } else if (!searchLoading && searchOrgNotExist) {
-      return (
-        <Panel
-          variant={PanelVariant.Error}
-          showIcon={false}
-          title={String(t('api_delegation.buisness_search_notfound_title'))}
-          forceMobileLayout={true}
-        >
-          <div>
-            {t('api_delegation.buisness_search_notfound_content')}{' '}
-            <a
-              className={classes.link}
-              href='https://www.brreg.no/'
-              target='_blank'
-              rel='noreferrer'
-            >
-              {t('common.broennoeysund_register')}
-            </a>
-          </div>
-        </Panel>
-      );
-    } else if (!searchLoading && promptOrgNumber) {
-      return (
-        <Panel
-          variant={PanelVariant.Info}
-          showIcon={false}
-          title={String(t('api_delegation.buisness_search_info_title'))}
-          forceMobileLayout={true}
-        >
-          {t('api_delegation.buisness_search_info_content')}
-        </Panel>
-      );
-    }
-  };
-
   return (
     <PageContainer>
       <Page
         color='dark'
         size={isSm ? 'small' : 'medium'}
       >
         <PageHeader icon={<ApiIcon />}>{t('api_delegation.give_access_to_new_api')}</PageHeader>
+        {chosenApis.length < 1 && (
+          <RestartPrompter
+            spacingBottom
+            restartPath={
+              '/' + ApiDelegationPath.OfferedApiDelegations + '/' + ApiDelegationPath.ChooseApi
+            }
+            title={t('common.an_error_has_occured')}
+            ingress={t('api_delegation.delegations_not_registered')}
+          />
+        )}
         <PageContent>
```

Comment:
Moving this outside of the pageContent makes the page look a bit strange imo
The gap of blue in the middle makes the page look broken, and the shadow that is still present as a border on that gap adds to that feeling.

The regular content is now also rendered unlike before, and while I'm not necessarily opposed to the principle of that idea, I wonder if it was intentional and whether you've taken into abbount if the info in the info-alert and the page content could contradict eachother and confuse the user?

![image](https://github.com/Altinn/altinn-access-management-frontend/assets/22012997/73c429eb-a571-450f-bf40-2f3a78fe86d8)